util/check-format-test.c

```diff
@@ -9,29 +9,66 @@
  * https://www.openssl.org/source/license.html
  */
 
-/* this demonstrates/tests cases where check-format.pl should complain */
+/* This demonstrates/tests cases where check-format.pl should complain */
+/* In total, 55 formattiing issues should be reported for this file */
```

Comment:
I‘m not sure whether it’s a good idea to hardcode the expected  number of  reported issues in a comment. People will forget to update it. Better make it a variable (expected_issues) and add a test condition for it. 

typo: formatting

Reply:
Indeed stating the number of expected issues cannot be the real solution.
I'll add a tag a per issue in the test input and these tags will be counted automatically.

Reply:
I've added the tags in `check-format-test.c` in a way such that the script can use them for self-tests.